lighthouse-core/audits/installable-manifest.js

```diff
@@ -5,38 +5,112 @@
  */
 'use strict';
 
-const MultiCheckAudit = require('./multi-check-audit.js');
-const ManifestValues = require('../computed/manifest-values.js');
+const Audit = require('./audit.js');
 const i18n = require('../lib/i18n/i18n.js');
+const ManifestValues = require('../computed/manifest-values.js');
 
+/* eslint-disable max-len */
 const UIStrings = {
   /** Title of a Lighthouse audit that provides detail on if a website is installable as an application. This descriptive title is shown to users when a webapp is installable. */
-  title: 'Web app manifest meets the installability requirements',
+  'title': 'Web app manifest and service worker meet the installability requirements',
   /** Title of a Lighthouse audit that provides detail on if a website is installable as an application. This descriptive title is shown to users when a webapp is not installable. */
-  failureTitle: 'Web app manifest does not meet the installability requirements',
+  'failureTitle': 'Web app manifest or service worker do not meet the installability requirements',
   /** Description of a Lighthouse audit that tells the user why installability is important for webapps. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
-  description: 'Browsers can proactively prompt users to add your app to their homescreen, ' +
-    'which can lead to higher engagement. ' +
-    '[Learn more](https://web.dev/installable-manifest/).',
+  'description': `Service worker is the technology that enables your app to use many Progressive Web App features, such as offline, add to homescreen, and push notifications. With proper Service Worker and manifest implementations, browsers can proactively prompt users to add your app to their homescreen, which can lead to higher engagement. [Learn more](https://web.dev/installable-manifest/).`,
```

Comment:
```suggestion
  'description': `Service worker is the technology that enables your app to use many Progressive Web App features, such as offline, add to homescreen, and push notifications. With proper service worker and manifest implementations, browsers can proactively prompt users to add your app to their homescreen, which can lead to higher engagement. [Learn more](https://web.dev/installable-manifest/).`,
```

> https://web.dev/installable-manifest

we'll need to update this to include the content from the other service-worker audits